### DISCUSSION TITLE
tutorial: Using SELECT Statements: core or ORM?

### DISCUSSION THREAD

**Discussion by mathause** [sqlalchemy/sqlalchemy#9450] [Usage Questions]
The whole page on [Using SELECT Statements](https://docs.sqlalchemy.org/en/20/tutorial/data_select.html#selecting-orm-entities-and-columns) is classified as core (dark blue) but there are considerable parts on using `Session` - shouldn't they be classified as ORM?

I find that a bit confusing. I am now not sure if I misunderstand something - which might very well be.

(I think this should be an issue but there seems no appropriate category for this kind of question/ bug, so I try here.)

**Comment by zzzeek** [comment]
front matter, this is the first comment I've ever gotten on "the blue bars" thing.    So one concern I have when I do these color coding types of things is one of accessibility; can people see "blue" at all?  how to make the "blue" visible but not intrusive?   

at the same time, if someone is not looking at blue bars at all for whatever reason, sections should also note explicitly what the target audience is.   So here the target is discussed at the top: "ORM Readers - the content here applies equally well to both Core and ORM use and basic ORM variant use cases are mentioned here. However there are a lot more ORM-specific features available as well; these are documented at [ORM Querying Guide](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)."

In this case, you are actually using "the blue bars" to give you direction, as I intended!  but the problem is there is actually a light blue bar on the left, it's just thinner.  both for page width and also symbolic of "ORM "uses" this but doesnt "own" it" .  but if you aren't noticing it I would maybe have to make it thicker.

the "blue bars on both sides" convention is noted in the front matter (which I understand most people probably skip) at https://docs.sqlalchemy.org/en/20/tutorial/index.html  meaning "A section that has both light and dark borders on both sides will discuss a Core concept that is also used explicitly with the ORM.".

so...that's the answer but maybe the left bar has to be made more noticeable.

**Reply by CaselIT** [reply]
should orm adopt another colour? like a green?

**Reply by zzzeek** [reply]
green , that is some real out of the box thinking!

**Reply by CaselIT** [reply]
I know right! I wonder how I came up with that :)

**Reply by zzzeek** [reply]
any re: green, I would make the blue a little thicker maybe, or maybe make it more of a pattern, not sure.   green might be fine but I'm not aesthetically motivated to play around with that right now

**Reply by CaselIT** [reply]
sure, I was just suggesting an alternative.

**Reply by zzzeek** [reply]
might be  great idea!   im being emotionally attached to blue at the moment.  I have the problem, not you
